src/query/queryBuilder.ts

```diff
@@ -165,8 +165,17 @@ function buildConditionExpressions<T extends object = AnyObject>(
             ` AND ${valueExpression}_end`,
         );
       } else if (operator === 'IN') {
-        expressionAttributeValues[valueExpression] = `(${condition.join(',')})`;
-        filterExpression.push(`${keyName} ${operator} ${valueExpression}`);
+        let expressionTemp = '';
```

Comment:
give me an example of what this does.

I can't get it by reading the code below

Reply:
example:

```
var params = {
    TableName: environment.ddbContentTableName,
    ProjectionExpression: "Title, ContentId, Link",
    FilterExpression: "ContentId in (:contentId1, :contentId2, :contentId3, :contentId4)",
    ExpressionAttributeValues: {":contentId1":102,":contentId2":104,":contentId3":103,":contentId4":101}
};
```